force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -5864,6 +5864,62 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Amount</shortDescription>
         <value>Amount</value>
     </labels>
+    <labels>
+        <fullName>commonPaymentMethod</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Payment Method</shortDescription>
+        <value>Payment Method</value>
+    </labels>
```

Comment:
I don't think we need to add a label for "Payment Method". Can we use the field label from the Recurring Donation object's Payment Method field instead?

Reply:
good suggestion